ios/FluentUI.Demo/FluentUI.Demo/Demos/ThemingDemoController.swift

```diff
@@ -190,3 +216,28 @@ open class CustomStyle: FluentUIStyle {
         }
     }
 }
+
+class CustomCardNudgeTokens: CardNudgeTokens {
+    let purplePrimary: ColorSet = ColorSet(light: 0x6227A7)
+    override var accentColor: ColorSet {
+        return purplePrimary
+    }
+    override var outlineColor: ColorSet {
+        return purplePrimary
+    }
+    override var subtitleTextColor: ColorSet {
+        return purplePrimary
+    }
+    override var textColor: ColorSet {
+        return purplePrimary
+    }
+    override var cornerRadius: CGFloat {
+        return 0.0
+    }
+}
+
+class CustomTokenFactory: TokenFactory {
```

Comment:
Have you checked requirements with the client apps (Teams/GroupMe/Skype) that need theme swapping whether the global and alias also need to reflect different values?